_components/header/header.html

```diff
@@ -92,85 +86,52 @@ <h2>Header usability</h2>
 <h2 class="usa-heading heading-margin-alt">Header accessibility</h2>
 <ul class="usa-content-list">
   <li>
-    Include skip navigation links to allow users with screen readers to bypass
-    long navigation lists. Make sure you include an id at the beginning of your
-    main content and that it matches the skipnav link. Find more <a href="http://webaim.org/techniques/skipnav/">information on these links</a> at WebAIM.
-
+    <strong>Add a skip navigation link before the header.</strong> Include skip navigation links to allow users with
+    screen readers to bypass long navigation lists. Make sure you include an <code>id</code> at the beginning of your
+    main content and that it matches the skip navigation link. Find more <a
+      href="http://webaim.org/techniques/skipnav/">information on these links</a> at WebAIM.
   </li>
   <li>
-    Include tab focus for all top-level navigation navigation items — this feature will
-    allow keyboard-reliant users to easily navigate interactive items.
+    <strong>Ensure your horizontal navigation is keyboard compatible.</strong> Test to make sure
+    users can use <code>Tab</code> to navigate and <code>Space</code> (or <code>Enter</code>) to open pages.
   </li>
   <li>
-    Ensure your horizontal navigation is keyboard compatible; test to make sure
-    users can use Tab to navigate and Space (or Enter) to open pages.
+    <strong>Include tab focus for all top-level navigation navigation items.</strong> This feature will
+    allow keyboard-reliant users to easily navigate interactive items.
   </li>
   <li>
-    Avoid using hover to expand dropdown lists. Hover is difficult for some
+    <strong>Avoid using hover to expand dropdown lists.</strong> Hover is difficult for some
     users and won't work on touch screens. Dropdowns should expand on click or
     with keyboard navigation.
   </li>
   <li>
-    The Design System uses semantic header and nav elements with
-    <code>role="banner"</code> and <code>role="navigation"</code> respectively.
-    <code>role="banner"</code> is your masthead.
-  </li>
-  <li>
-    You can use multiple nav elements for groups of navigation links, but you
-    should only use one <code>role="navigation"</code> for the main nav of a
-    page.
```

Comment:
> **Note**
> Replaced this section with the "Add context by labelling your nav element." section. We do not use the navigation role in our code, and it seems unnecessary since we are using the semantic `nav` element. Is there a reason we would want to redundantly declare the role? Should we keep this note in but specify it is only if you are not using the `nav` element?